mock/imposter-config.yaml

```diff
@@ -0,0 +1,24 @@
+plugin: openapi
+specFile: https://mongodb-mms-prod-build-server.s3.amazonaws.com/openapi/acc6ed9d3dd14f4b74d68486bf49c906b3cf7318-v1.json
+
+validation:
+  request: true
+  levels:
+    validation.request.body.missing: ERROR
```

Comment:
enabling the request validation based on the validation field in the open API spec